nvidia_nims/vista_3d_remote_nim.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !python -c "import pyvista" || ipywidgets 'pyvista[all,trame]'

</pre><p> </p><p>Looks like a typo? Should be <code>!python -c "import pyvista" || pip install -q 'pyvista[all,trame]'</code></p><p> </p><p>BTW, pyvista 0.44.2 does not provide the extra 'trame', "all" should be enough here?</p><p><code>WARNING: pyvista 0.44.2 does not provide the extra 'trame'</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed.